build.gradle

```diff
@@ -21,10 +21,11 @@ checkstyle {
 
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'    
 	runtimeOnly 'mysql:mysql-connector-java'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 }
 
-task checkstyle {
+task check {
```

Comment:
Looks like we've got a tabs vs. spaces issue here. Let's pick one, and reformat the entire file using that style. I'm not picky which one it is.